src/tests/mock/mockData.js

```diff
@@ -1,9 +1,10 @@
 import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
 
 dotenv.config();
 
 const { passwordEnv } = process.env;
-
+const userToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MywiZW1haWwiOiJqYW1zb24xQGdtYWlsLmNvbSIsInVwZGF0ZWRBdCI6IjIwMTktMTEtMTVUMDg6NDQ6MjUuODI4WiIsImNyZWF0ZWRBdCI6IjIwMTktMTEtMTVUMDg6NDQ6MjUuODI4WiIsImlzVmVyaWZpZWQiOm51bGwsInNpZ251cFR5cGUiOm51bGwsImlhdCI6MTU3MzgwNzQ2NX0.a_OQFNggkrmAObYfr6bUOe4NTPYdYnLRToCVatDMsTk';
```

Comment:
Call the generate token function and make it generate a token instead of having this long line.

Reply:
For test purposes, you should log a user in in a before block, save the token and use the variable while you work. 

What you have done has many downsides, one being that the token can expire and tests start failing. Another is you are exposing our token and inviting malicious attacks.

Reply:
I have corrected it. Thanks for this feedback